mir_eval/util.py

```diff
@@ -138,6 +138,8 @@ def interpolate_intervals(intervals, labels, time_points, fill_value=None):
         The ``i`` th interval spans time ``intervals[i, 0]`` to
         ``intervals[i, 1]``.
 
+        Intervals are assumed to be disjoint.
```

Comment:
Those are two different things.

Currently, the interpolator assumes that intervals are disjoint and cover some range `[t_min, t_max]`; the fill value gets applied outside the interval.

Disjointness is required for the labeling process to be a function, so I just made that requirement explicit in the docstring.

In the future, I see no reason to require full coverage in general, ie, a gap in the middle should be fine.  However, at present it will do the wrong thing here and just propagate the label from the interval preceding the gap.